src/mlx/warnings.py

```diff
@@ -319,6 +332,8 @@ def warnings_wrapper(args):
     parser.add_argument('-v', '--verbose', dest='verbose', action='store_true')
     parser.add_argument('--command', dest='command', action='store_true',
                         help='Treat program arguments as command to execute to obtain data')
+    parser.add_argument('--ignore-retval', dest='ignore', action='store_true',
```

Comment:
i would think the default is not to ignore the return value, so `store_false`?

Reply:
I see that as a request to ignore-return value - which is considered a exceptional case (otherwise we do not ignore and that is why it is True. Also checks check if we should ignore return value (true) and not the other way around :smile: What would a flagname be to program to ignore a return value to make it false?

Reply:
there is a misunderstanding with the default option from my side, i verified your implementation and it does what we both described above :+1: